game.js

```diff
@@ -1,4 +1,6 @@
 function gameEntryPoint() {
+    new GTheme().apply();
```

Comment:
Tenhle soubor není vůbec potřeba upravovat, to vyřeší web v dalším updatu, na kterým pracuji

Reply:
Odstraníš to prosím aji v téhle hře?